iac-extensions/cloudformation/src/main/java/org/sonar/iac/cloudformation/checks/PrivilegeEscalationCheck.java

```diff
@@ -69,48 +81,23 @@ private static List<SecondaryLocation> secondaryLocations(Statement statement, P
     List<SecondaryLocation> secondaryLocations = new ArrayList<>();
     String actionsMsg = vector.getPermissions().size() == 1 ? String.format(MESSAGE_ACTION_SINGLE, vectorName) :
       String.format(MESSAGE_ACTION_MULTIPLE, vectorName);
+    addSecondaryLocationsFromAction(statement, vector, vectorName, secondaryLocations, actionsMsg);
+    addSecondaryLocationsFromResource(statement, secondaryLocations);
+    return secondaryLocations;
```

Comment:
See comment on this method in the Terraform check. I suggest doing the same changes here too.